### DISCUSSION TITLE
Formatting output in interactive command

### DISCUSSION THREAD

**Discussion by helander** [pocketbase/pocketbase#3756] [Q&A] [answered]
I have made a command hook to add an interactive command in Javascript. I can use console.log() to have the command print out information. Each printed record is pre-pended with a date & time string. Is there some way to print stuff without having the date & time string pre-pended?

**Comment by ganigeorgiev** [comment]
No, I don't think the timestamp fields can be removed with the JS hooks unless modifying the goja `console` implementation to not use the Go `log` package defaults.

A workaround could be to try to invoke the `echo` shell command but it may not work everywhere.

I've added a task in the roadmap to consider eventually exposing some of the Go stdout/stderr write primitives with the next release.

**Reply by ganigeorgiev** [reply, marked as answer]
Actually if you are in a command hook context, you can try to make use of the [print/printf/etc. Command methods](https://pkg.go.dev/github.com/spf13/cobra#Command.Print):

```js
$app.rootCmd.addCommand(new Command({
    use: "hello",
    run: (cmd, args) => {
        cmd.print("something")
    },
}))
```
(_JSVM reference to the available `cobra.Command` methods - https://pocketbase.io/jsvm/interfaces/cobra.Command.html_)
